README.md

```diff
@@ -282,6 +282,8 @@ dom.window.ran === 3;
 
 This is somewhat-advanced functionality, and we advise sticking to normal DOM APIs (such as `window.eval()` or `document.createElement("script")`) unless you have very specific needs.
 
+`runVMScript` also takes an `options` object as its second argument. See the [Node docs](https://nodejs.org/api/vm.html#vm_script_runincontext_contextifiedsandbox_options) for details.
```

Comment:
Some nits:

- I've tried to stick with adding parens for function/method names, so `runVMScript()`.
- I've tried to stick to Node.js instead of Node.

Also maybe we should update the header to `runVMScript(script[, options])`? Dunno, could go either way.